rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -358,9 +358,14 @@ class DSCTechniqueWriter(
                 ) ( e =>
                   IOError(s"Could not find dsc Technique '${technique.name}' in path ${basePath}/${techniquePath}", Some(e))
                 )
+      // Powershell files needs to have a BOM added at the beginning of all files
+      contentWithBom : List[Byte] =
+        // Bom, three bytes: EF BB BF
```

Comment:
Can you add a link towards the spec/convention/wikipedia/etc? If one day we want to change to something else?